src/server/services/vergunningen.ts

```diff
@@ -156,6 +172,22 @@ export type VergunningenSourceData = {
   status: 'OK' | 'ERROR';
 };
 
+type NotificationItems = Exclude<
+  Vergunning,
+  | TVMRVVObject
+  | Vakantieverhuur
+  | BBVergunning
+  | ERVV
+  | VakantieverhuurVergunningaanvraag
+  | GPP
+  | EvenementMelding
+  | Omzettingsvergunning
+>;
+
+type NotificationLinks = {
+  [key in NotificationItems['caseType']]: string;
```

Comment:
So you can make it less restrictive by doing: `[key in Vergunning['caseType']]?: string;` Saves you maintenance